ModbusScope.pro

```diff
@@ -4,5 +4,5 @@ TEMPLATE = subdirs
 CONFIG += ordered
 
 SUBDIRS += \
-    src \
+    src
```

Comment:
Why removing " \ ". Isn't it required?

Reply:
If you don't want to be nagged by the tests it is required :smile:

Reply:
Fixed in de427d5